README.md

```diff
@@ -1,22 +1,76 @@
-# TITLE 
-[![All Contributors](https://img.shields.io/badge/all_contributors-1-orange.svg?style=flat-square)](#contributors)
+<div align="center">
+<h1>FLINT UI</h1>
+<p>
+UI for FLINT.Cloud project which aims to build a continuous deployment pipeline to offer FLINT as a SaaS overcloud
+</p>
+<a href="#contributors"><img src="https://img.shields.io/badge/all_contributors-1-74e8a3.svg?style=flat-square" /></a>
+<a href="https://github.com/moja-global/FLINT-UI/network/members"><img src="https://img.shields.io/github/forks/moja-global/FLINT-UI?color=74e8a3&style=flat-square" /></a>
+<a href="https://github.com/moja-global/FLINT-UI/stargazers"><img src="https://img.shields.io/github/stars/moja-global/FLINT-UI?color=74e8a3&style=flat-square" /></a>
+<a href="https://github.com/moja-global/FLINT-UI/blob/master/LICENSE"><img src="https://img.shields.io/github/license/moja-global/FLINT-UI?color=74e8a3&style=flat-square" /></a>
+</div>
 
-Text 
+## About Project
 
-## How to Get Involved?  
+This project is UI and dashboard for [FLINT.Cloud](https://github.com/moja-global/FLINT.Cloud) project which aims to build a continuous deployment pipeline to offer FLINT as a SaaS overcloud. The project also aims to simplify the process of installation by supporting a single command or step installation process.
```

Comment:
Change to "This project provides an intuitive way for new to explore some preconfigured FLINT modules, including the Generic Budget Carbon Model (GCBM), in order to better understand how the FLINT system works. Our client is written as a Web application and can be used in a local or remote environment. Please contact us if you'd like help deploying your own cloud instance or customising the client."